rotkehlchen/api/v1/resources.py

```diff
@@ -1577,6 +1584,49 @@ def post(
             async_query=async_query,
         )
 
+    @use_kwargs(put_schema, location='json')
+    def put(
+        self,
+        from_asset: Asset,
+        to_asset: Asset,
+        price: Price,
+        timestamp: Timestamp,
+    ) -> Response:
+        return self.rest_api.add_manual_price(
+            from_asset,
```

Comment:
nitpick: be using kwargs. Why? If you messup the argument order the python itnerpreter CAN NOT know it. But if you use kwargs, order does not matter, and if you miss an argument the interpreter catches it.